pinc/projectinfo.inc

```diff
@@ -9,8 +9,7 @@ function Project_getNumPagesInState( $projectid, $page_state , $counter = "*")
     $res = mysqli_query(DPDatabase::get_connection(), "SELECT COUNT($counter) FROM $projectid WHERE state='$page_state'");
     if (!$res)
     {
-        echo mysqli_error(DPDatabase::get_connection());
-        return 0;
+        die(DPDatabase::log_error());
```

Comment:
You're changing the behavior from `return 0` to `die`. I'd advise against, unless you've convinced yourself that it's appropriate.

Reply:
Good call - fixed.